6.1.Числовые_типы_данных_int_float/6.1.12.py

```diff
@@ -0,0 +1,9 @@
+num1 = int(input())
+num2 = int(input())
+num3 = int(input())
+num4 = int(input())
+num5 = int(input())
+nummin = min(num1, num2, num3, num4, num5)
+nummax = max(num1, num2, num3, num4, num5)
+print(f"Наименьшее число = {nummin}")
+print(f"Наибольшее число = {nummax}")
```

Comment:
```suggestion
print(f"Наименьшее число = {nummin}", f"Наибольшее число = {nummax}", sep="\n")
```

Тут можно обойтись одним вызовом функции `print()`.